src/main/webapp/WEB-INF/pages/footer.jsp

```diff
@@ -0,0 +1,36 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<jsp:useBean id="recentlyViewed" type="java.util.LinkedList" scope="request"/>
+
+<h3>Recently viewed</h3>
```

Comment:
Please don't display title and empty table if there are no recently viewed products.

It actually shows an empty recently viewed table for me here: http://localhost:8080/phoneshop-servlet-api/products

And on PDP it shows only one current product as recently viewed.